tests/foreman/api/test_contentview.py

```diff
@@ -361,6 +355,7 @@ def test_positive_restart_publish_via_dynflow(self):
         """
 
 
+@pytest.mark.usefixtures('module_org')
```

Comment:
Actually, is there a reason for each class to have their own organization? Perhaps you could speed it up further by using one organization everywhere?

Reply:
Probably to avoid the clash of entities/actions that affect the asserts of tests. Its been followed since long and when we developed robottelo.

Reply:
"We have always done it this way" is not a very good reason, but I understand you don't want to touch too many things at one time.